networkconfig/holesky-stage.go

```diff
@@ -16,7 +16,7 @@ var HoleskyStage = NetworkConfig{
 	GenesisEpoch:         1,
 	RegistrySyncOffset:   new(big.Int).SetInt64(84599),
 	RegistryContractAddr: "0x0d33801785340072C452b994496B19f196b7eE15",
-	AlanForkEpoch:        99999999,
+	AlanForkEpoch:        0,
```

Comment:
do we want to merge it to be post-fork?

Reply:
@y0sher just testing deployment, will revert soon